OEPS/OEP-35.mediawiki

```diff
@@ -0,0 +1,150 @@
+<pre>
+  OEP: 35
+  Title: Ontology Cross-Chain Standard
+  Author: zhangmh<siovanus@126.com>
+  Type: Standard
+  Status: Accepted
+  Created: 2019-6-12
+</pre>
+
+=Abstract=
+The OEP-35 Proposal is a standard interface for Ontology Cross-Chain, this standard allows you to transfer asset or any type of informations in smartcontract among different homogeneous sidechains and heterogeneous side-chains.
+
+=Motivation=
+Now blockchains are isolated and unable to communicate with each other. But with the development of blockchain ecosystem and smart contract DApp, the requirement of breaking through barriers between different block chains is more and more intense.
+
+Ontology provides multichain network and allows ontology mainnet and these side-chains communicate to each other. Developers can use cross-chain API in their ontology smartcontract.
+
+The OEP-35 proposal is the standard specify how the this technology works.
+
+=Specification=
+==Contract==
+===Chain Manager Contract===
+====initConfig====
+<source lang="go">
+func InitConfig(native *native.NativeService) ([]byte, error) {}
+</source>
+This method init chain manager contract, will be invoked automatically when block chain starts.
+
+====registerMainChain====
+<source lang="go">
+func RegisterMainChain(native *native.NativeService) ([]byte, error) {}
+</source>
+This method is available in side chain, register main chain's key header to side chain.
+
+====registerSideChain====
+<source lang="go">
+func RegisterSideChain(native *native.NativeService) ([]byte, error) {}
```

Comment:
`native *native.NativeService` does not expose any valid information, need list the actual param and return type like other OEPs.